tests/testpeeridauth.nim

```diff
@@ -0,0 +1,57 @@
+{.used.}
```

Comment:
tests that do interact with external api, should be part for integration tests group.